### DISCUSSION TITLE
Maximum number of running instances reached

### DISCUSSION THREAD

**Discussion by phlegx** [jertel/elastalert2#195] [Q&A] [answered]
Hi there!

I try to auto resolve my alerts for Pagerduty, but it seems that Pagerduty cannot be called sometimes when it tries to  autoresolve the alert. I get this Warming:

```
WARNING:apscheduler.scheduler:Execution of job "ElastAlerter.handle_rule_execution (trigger: interval[0:00:10], next run at: 2021-05-27 12:17:58 UTC)" skipped: maximum number of running instances reached (1)
```

Restarting the container then usually resolves this and the alert in Pagerduty then gets resolved and the `warning` disappears.

Any ideas what might is causing that? 

thanks in advance
Martin

**Comment by jertel** [comment, marked as answer]
Rules are prevented from executing concurrently. That warning is telling you that the same rule is already being executed in another scheduler thread. I see your interval is set to a very short 10 seconds. Is it possible that the pagerduty alerts are taking more than 10 seconds to execute?
